addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -295,7 +339,8 @@ function factory(dependencies) {
          * hint `message-received`.
          */
         hasAutoScrollOnMessageReceived: attr({
-            default: true,
+            compute: '_computeHasAutoScrollOnMessageReceived',
+            default: false,
```

Comment:
default not really needed with compute (except very specific situation where it's used as part of another compute that is computed first and for which having the value undefined would be an issue)